src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -166,7 +166,7 @@ public static function fromUrlEncodedString($serializedConfiguration)
             }
         }
 
-        if (isset($normalizedConfiguration['disabled'])) {
+        if (isset($normalizedConfiguration['disabled']) && '0' !== $normalizedConfiguration['disabled']) {
```

Comment:
let's use `filter_var($normalizedConfiguration['disabled'], FILTER_VALIDATE_BOOLEAN)`?
the same concern applies to "verbose" also

Reply:
I updated with `filter_var`. To avoid adding extra complexity, this kind of configuration `SYMFONY_DEPRECATIONS_HELPER=...&disabled` won't disable deprecations anymore. But maybe we can consider that supporting this kind of config is not relevant ? (However `SYMFONY_DEPRECATIONS_HELPER=disabled` and `SYMFONY_DEPRECATIONS_HELPER=...&disabled=1` still disable deprecations)

Reply:
> SYMFONY_DEPRECATIONS_HELPER=...&disabled won't disable deprecations anymore

that's an issue, better not breaks things, let's figure out a way to keep it working

Reply:
Can you confirm that you prefer to use `filter_var` and add little extra complexity over using `'0' !== $normalizedConfiguration['disabled']` like it was initially done ?

Reply:
I made a try :)